src/mscorlib/src/System/PinnableBufferCache.cs

```diff
@@ -4,31 +4,82 @@
 #define ENABLE
```

Comment:
We are not taking nice to have cleanups anymore at this point. This formatting, etc. cleanup should wait for 2.2.

Reply:
Fine